src/ansys/motorcad/core/methods/rpc_methods_general.py

```diff
@@ -458,3 +458,45 @@ def quit(self):
     def set_free(self):
         """Free the Motor-CAD instance."""
         return self.connection._set_free()
+
+    def download_mot_file(self, file_path):
```

Comment:
If the file i want to download has some error messages, which i okayed. However as soon as i use this funtion the same error message popup and process stops. image.png

Reply:
Hi @ravi-ansys, I think this is just a Motor-CAD warning. This works fine when MessageDisplayState=2